packages/public-docsite-setup/package.json

```diff
@@ -25,7 +25,9 @@
   "license": "MIT",
   "devDependencies": {
     "@fluentui/eslint-plugin": "^1.1.0",
-    "@fluentui/scripts": "^1.0.0"
+    "@fluentui/scripts": "^1.0.0",
+    "rollup": "^2.7.6",
```

Comment:
@chrisdholt Just FYI, I'm adding some very simple usage of rollup in v8 as part of this PR (see next file for details). Since web-components was previously the only thing in the repo using rollup, I went ahead and used the same versions as you have to avoid duplicates, even though they're a little outdated. Whenever you eventually update rollup for web-components, it should be safe and straightforward to go ahead and update this usage as well.